### DISCUSSION TITLE
全局拦截器中，当响应出现异常，如何在全局关闭loading

### DISCUSSION THREAD

**Discussion by AnyFork** [alovajs/alova#291] [Q&A]
### 请详细描述bug

全局拦截器中，当响应出现异常，如何在全局关闭loading

### 复现步骤

无

### alova 版本

2.16.0

**Comment by JOU-amjs** [comment]
问题类请在Discussion提问

问题分两类。
1. 请求状态码异常，如果你使用的是GlobalFetch适配器，也会触发success响应钩子，你可以这样做
```js
createAlova({
  responded(response) {
    if (response.status > 200) {
      // 处理异常，比如关闭loading
    }
  }
})
```

2. 断网或超时，触发error响应钩子，你可以这样做
```js
createAlova({
  responded: {
    // ...
    onError(response) {
      // 处理异常，比如关闭loading
    }
  }
})
```

> 注意，如果你使用的axios适配器，依照axios特性，状态码异常也会进入error钩子

**Reply by AnyFork** [reply]
全局拦截器：
![image](https://github.com/alovajs/alova/assets/49142023/78d7cc31-34b0-4d86-a0d7-0cacdae28600)
如何关闭useRequest 中的loading

**Comment by JOU-amjs** [comment]
useRequest中的loading不需要你自己关闭，不管成功与否会自己关闭。
可能是我还没有理解你的情况，请提供一些更详细的情况描述和相关代码，我才可以更好理解

**Reply by AnyFork** [reply]
useWatcher 在接口超时情况下不会自动关闭loading

**Reply by MeetinaXD** [reply]
这边测试了一下 useWatcher 这个接口。在单纯接口超时引发 error 的情况下，loading 是能够自动变为 false 的。

请问你这边能否提供一下能复现问题的最小单元代码？
